Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,6 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics * cms.SequencePlaceholder("mix"))
```

Comment:
Indeed, which makes the change a bit odd... anyway, its not consistent with what ConfigBuilder expects. So, either it should be reverted or ConfgBuilder should be modified to expect (pre)validationCosmics instead of (pre)validation in the case someone decides to run the validation sequence in a cosmics workflow.